cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -1,137 +1,138 @@
 # -*- coding: utf-8 -*-
-import cfme.intelligence.chargeback.assignments as cb
 import pytest
 import random
 
-from cfme.intelligence.chargeback.assignments import AssignmentsView
 from cfme import test_requirements
-from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigate_to, navigator
 
 pytestmark = [
     pytest.mark.tier(3),
     test_requirements.chargeback
 ]
 
 
+@pytest.fixture
+def assigns_collection(appliance):
+    return appliance.collections.assignments
+
+
 @pytest.mark.meta(blockers=[1273654])
-def test_assign_compute_enterprise(appliance, virtualcenter_provider):
```

Comment:
why is virtualcenter_provider fixture removed from the parameters for this test?

Reply:
That's probably a mistake on my part. I'll add it back right away

Reply:
@mshriver Actually, I just realized that in Pycharm, I am being shown that this parameter is not being used anywhere and that is why I had removed it. It still shows the same warning/note. Should I leave it as it is or shall I delete it?

Reply:
I see - yep that is expected and not a problem when it comes to pytest methods and fixtures.  This is due to the nature of fixture methods - they're uniquely different from normal functions in python.

When a fixture is included in a pytest method parameters, that fixture is executed, regardless of its use in the test method.

Pycharm isn't so smart about this, and sees that you have a parameter to the function that isn't used anywhere within scope. If you take a look around our test library, you'll see this highlighting in plenty of places.

Even though the fixture name may not be used within scope (consider `setup_provider` fixture), its still necessary to have it defined as a parameter to the method - so that it will be executed during pytest setup phase for the method.

Reply:
Oh I see , silly PyCharm :p